src/mixins/createFormFieldMixin.ts

```diff
@@ -134,6 +168,64 @@ export function stringToValue(str: string): any {
 	}
 }
 
+export function getFormFieldNodeAttributes(state: any): VNodeProperties {
+	const allowedAttributes = ['value', 'type', 'checked', 'descriptionID', 'disabled', 'inputmode', 'maxlength', 'minlength', 'multiple', 'name', 'placeholder', 'readonly', 'required', 'invalid'];
+	const nodeAttributes: any = {};
+
+	for (const key in state) {
```

Comment:
Also why can this not just be a `nodeAttribute` function on the instance rather than an exported function?